examples/csrApproval/csrApproval.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
```

Comment:
you can take a look at other examples, there are some shared values defined in `Directory.Build.props`